translations/strings.yaml

```diff
@@ -697,6 +697,21 @@ inAppMessaging:
   deleteMessage:
     value: Delete message
     comment: An action to delete a message from the users inbox
+  warningBadge:
+    value: Warning
+    comment: A badge shown in a message signifying that this is a warning message
```

Comment:
Are these actual text badges or alt text for images/icons?

Reply:
They are badges with text:

<img width="68" alt="Screen Shot 2022-09-04 at 1 35 57 PM" src="https://user-images.githubusercontent.com/15353801/188326258-9c5fded4-d7da-4e76-bb56-a371d56372ab.png">
<img width="68" alt="Screen Shot 2022-09-04 at 1 36 35 PM" src="https://user-images.githubusercontent.com/15353801/188326280-211a7d6c-4413-4d66-923b-6900a56f5650.png">

Reply:
How much space is there? For example, "New update" would be "Nuovo aggiornamento" in Italian (in French update is `Mise à jour`, in Spanish `Actualización`).

Reply:
There should be plenty of space for those examples

 <img width="363" alt="Screen Shot 2022-09-06 at 2 24 03 PM" src="https://user-images.githubusercontent.com/15353801/188710516-97fab287-5f45-48ec-916e-67646a940537.png">

But if the string gets really long (which is technically possible), this would be the current UI - but I am looking for ways to improve that

<img width="353" alt="Screen Shot 2022-09-06 at 2 25 29 PM" src="https://user-images.githubusercontent.com/15353801/188710881-3acfd58a-2b83-4fa3-bb24-1aaae4a404e2.png">

Reply:
Thanks, wasn't sure if they would show up near the title, not on a separate line.

Reply:
Badge text is now elided 

<img width="360" alt="Screen Shot 2022-09-06 at 5 55 10 PM" src="https://user-images.githubusercontent.com/15353801/188755676-22d91472-6f5e-4e49-9e8c-767bc7e4e4c2.png">